### ISSUE TITLE
meshio for mesh I/O

### ISSUE THREAD

**Issue by clbarnes** [schlegelp/navis#51]
Spotted some commits implementing more mesh formats - would it be worth just (optionally?) depending on meshio and calling it a day? Improvements and additional formats could be contributed back upstream. Reducing the number of implementations out in the wild is probably for the best.

**Comment by schlegelp** [comment]
Ah good point! I didn't bother hunting for existing implementations for neuroglancer's "precomputed" format. I have independent functions in various other packages and this was actually my attempt at consolidating those into navis. 

At a glance it looks like `meshio` currently only deals with the legacy mesh format. The recent commits to `navis` do a bit more than that:
- read/write precomputed skeletons (including radii)
- (optionally) write manifest files for meshes
- (optionally) read/write `info` files

I guess the latter two could be contributed back upstream to `meshio`? As things stand, adding `meshio` as dependency would shed ~20 lines of code in `navis`.

**Comment by clbarnes** [comment]
Yes, it was a while ago that I added that so not surprised if the format as come on since then. As it is such a simple format, it also doesn't surprise that the dependency (which is pretty light without extras) wouldn't save much code, but it would add quite a lot of flexibility to navis for reading and writing different mesh formats.

**Comment by schlegelp** [comment]
Oh, I see. We might be thinking of slightly different things here. I'm not at all opposed to adding it as a dependency but perhaps you can elaborate a bit on how you would suggest proceeding?
